includes/attendee/attendee-repository.php

```diff
@@ -128,4 +128,46 @@ function ( object $row ) {
 			$rows
 		);
 	}
+
+	/**
+	 * Add a user as host of an event.
+	 *
+	 * @param Attendee $attendee The attendee to add as host.
+	 * @return void
+	 */
+	public function add_host( Attendee $attendee ): void {
+		$this->update_host( $attendee, 1 );
+	}
+
+	/**
+	 * Remove a user as host of an event.
+	 *
+	 * @param Attendee $attendee The attendee to remove as host.
+	 * @return void
+	 */
+	public function remove_host( Attendee $attendee ): void {
+		$this->update_host( $attendee, 0 );
+	}
+
+	/**
+	 * Update the host status of an attendee.
+	 *
+	 * @param Attendee $attendee The attendee to update.
+	 * @param int      $is_host  Whether the attendee is a host.
+	 * @return void
+	 */
+	private function update_host( Attendee $attendee, int $is_host ) {
```

Comment:
Similarly to my other comment about `add_host()`, I think we should have a generic `update_attendee( Attendee $attendee )` method that updates attendees, regardless of the attendee being a host or not.

Reply:
Maybe the best approach is to have:
- a `add_host( Attendee $attendee )` method.
- a `remove_host( Attendee $attendee )` method.
- add a new `update_attendee( Attendee $attendee )`, used by the previous methods, removing the `update_attendee( Attendee $attendee )` method.

What do you think about?

Reply:
Mark as resolved, because I have implemented [this suggestion](https://github.com/WordPress/wporg-gp-translation-events/pull/200#discussion_r1559642033).